integrations/privacy/osano.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: "Osano"
+---
+
+Add the following to your `mint.json` file to add the [Osano](https://www.osano.com/) cookie consent manager.
+
+<CodeGroup>
+
+```json Integration options in mint.json
+"integrations": {
+    "osano": "SOURCE"
+}
+```
+
+```json Example
+"integrations": {
+    "osano": "https://cmp.osano.com/2sUB2dqwqdkks/8dqwd-dwd86£-4a9b/osano.js"
+}
+```
+
+</CodeGroup>
+
+The `SOURCE` can be found as the `src` value in the code snippet generted by Osano.
+
```

Comment:
Can you add a Note that it has to start with https://cpm... so it's super crystal clear?